### ISSUE TITLE
clarification question

### ISSUE THREAD

**Issue by moeller0** [jeverley/dscpclassify#7]
The readme says:
These connections are classified as High-Throughput (AF13) by default and therefore have a higher drop probability than regular traffic in the Best Effort layer-cake tin.

How is that implemented? In cake all priority tiers >= besteffort have more or less the same drop probability as far as I can see.

**Comment by jeverley** [comment]
Yes you're right, my oversight there - it's only lower priority if using diffserv8, which most (including myself) don't use.
